automation/run-tests.sh

```
@@ -152,6 +153,18 @@ function run_tests {
             ${nmstate_pytest_extra_args}"
     fi
 
+    if  [ $TEST_TYPE == $TEST_TYPE_INTEG_KERNEL ];then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)